### ISSUE TITLE
Attempt to create storagegroup fails

### ISSUE THREAD

**Issue by vmorris** [zhmcclient/zhmccli#129]
### Actual behavior
`zhmc storagegroup create` results in the following stack trace:

```
Traceback (most recent call last):
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/bin/zhmc", line 33, in <module>
    sys.exit(load_entry_point('zhmccli==0.20.0.dev1', 'console_scripts', 'zhmc')())
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/zhmccli/_cmd_storagegroup.py", line 166, in storagegroup_create
    cmd_ctx.execute_cmd(lambda: cmd_storagegroup_create(cmd_ctx, options))
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/zhmccli/_helper.py", line 289, in execute_cmd
    cmd()
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/zhmccli/_cmd_storagegroup.py", line 166, in <lambda>
    cmd_ctx.execute_cmd(lambda: cmd_storagegroup_create(cmd_ctx, options))
  File "/home/vmorris/git/github.ibm.com/vmorris/zhmc-scripts/venv/lib64/python3.9/site-packages/zhmccli/_cmd_storagegroup.py", line 424, in cmd_storagegroup_create
    email_to_addresses = options['email_to_address']
KeyError: 'email_to_address'
```

### Expected behavior
Storage group is created

### Execution environment

* zhmccli version: 0.20.0.dev1
* zhmcclient version: 0.28.0
* Operating system (type+version): Fedora 33
* HMC version: 2.14.1
* CPC version: z14 (DPM)

**Comment by vmorris** [comment]
In fact, any attempt to create a storagegroup fails... this results in the same stack trace as above:

`zhmc storagegroup create --name trashme --cpc P0026A48 --type fcp`

**Comment by andy-maier** [comment]
Fixed in PR #133, targeted for zhmccli version 0.20.0

**Comment by andy-maier** [comment]
@vmorris Vance, the PR has been merged to master. Feel free to try it out.
